distribution/src/resources/drill-override-example.conf

```diff
@@ -214,7 +214,7 @@ drill.exec: {
 }
 
 # Below SSL parameters need to be set for custom transport layer settings.
-javax.net.ssl {
+ssl{
   keyStore: "/keystore.file",
```

Comment:
Can you look into this paul? This an example I got. 
SSL certificate keystore location.
Default: conf/default.keystore.
ssl.keystore.path can be configured in 2 ways:
By using the relative path, for example, ssl.keystore.path=conf/default.keystore, where the root directory is <Worklight Root Directory>\server\.
By using the absolute path, for example, ssl.keystore.path=/opt/wl/default.keystore for UNIX, or C:\\wl\\default.keystore for Windows.